x-pack/plugins/timelines/public/hooks/use_status_bulk_action_items.tsx

```diff
@@ -48,6 +28,57 @@ export const useStatusBulkActionItems = ({
   onUpdateFailure,
 }: StatusBulkActionsProps) => {
   const { updateAlertStatus } = useUpdateAlertsStatus();
+  const { addSuccess, addError, addWarning } = useAppToasts();
```

Comment:
toad management encapsulated in the public component to prevent duplication. Still calls optional `onUpdateSuccess` and `onUpdateFailure` when needed